docs/DeveloperGuide.md

```diff
@@ -494,20 +489,8 @@ testers are expected to do more *exploratory* testing.
 
 ## **Appendix: Planned Enhancements**
 
+Team size: 5
```

Comment:
Requirement gotten from: https://nus-cs2103-ay2425s1.github.io/website/admin/tp-w11.html

We have 10 planned enhancements as a shield.